src/debugger/assert.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#define ASSERT_MESGBUF_SIZE 256
```

Comment:
Eh, I would keep this doubled, just in case. We don't exactly have a crash screen for the crash screen if we somehow end up with a huge message (though it is admittedly not likely to happen).

Reply:
There's an argument to be made here that this could be repurposed for general use. Maybe that's better for a hacker to decide, but there's never gonna be a situation where this is used before a crash is already happening.